packages/gatsby-plugin-offline/README.md

```diff
@@ -166,6 +166,28 @@ const options = {
 }
 ```
 
+You can also use [Workbox CLI](https://developers.google.com/web/tools/workbox/modules/workbox-cli) to generate this file automatically. 
+
+```
+workbox wizard
+? What is the root of your web app (i.e. which directory do you deploy)? public/
```

Comment:
In order to have a public folder you need to have run `gatsby build` at least once. Would be good to note that's necessary for this setup.